docs/src/main/paradox/client-side/configuration.md

```diff
@@ -0,0 +1,51 @@
+# Configuration
+
+HTTP client settings are split into different sections  
+ 
+ * `akka.http.client`: basic client settings
+ * `akka.http.host-connection-pool`: pool settings
+ 
+## Basic Client Settings
+
+These settings influence the basic library behavior for each HTTP connection. When changed in the application's
+`application.conf` they globally affect the default client behavior.
+
+Basic client settings can be overridden in multiple ways:
+
+ * by passing custom @apidoc[ClientConnectionSettings] instances to APIs in @scala[@apidoc[Http$]]@java[@apidoc[Http]]
+ * by overriding settings in `akka.http.host-connection-pool.client`, these overrides will take effect whenever a pool is used
+   like with `Http().singleRequest`
+ * by putting custom @apidoc[ClientConnectionSettings] into @apidoc[ConnectionPoolSettings] and passing those to APIs in `Http`
+ * by using [per-host overrides](#per-host-overrides)
+
+@@snip [reference.conf]($akka-http$/akka-http-core/src/main/resources/reference.conf) { #client-settings }
```

Comment:
Paradox should already understand `/` as the project root, so the `$akka-http$` prefix is no longer needed (and makes the link work when viewing the markdown in the github ui)

```suggestion
@@snip [reference.conf](/akka-http-core/src/main/resources/reference.conf) { #client-settings }
```

Reply:
Nice as well, also copied from somewhere else :)

Reply:
Fixed all of those in #3339.